connect_db.php

```diff
@@ -1,11 +1,37 @@
-<?php
-function connect_db()
-{
-$db = new mysqli();
-$test = $db->connect("localhost","root","budget_123","moneycontribution");
-$sql = "SELECT * FROM `user_profile_required`";
-$data = $db->query($sql);
-$count = $data->num_rows;
-$db->close();
-}
-?>
\ No newline at end of file
+<?php 
+ function select($query){
```

Comment:
Can you please create one more function which will use to connect with database and return $db and we can use that $db in these all functions and there is only one place where we have all these information (username, password etc).